CODENAMES.md

```diff
@@ -0,0 +1,12 @@
+This file contains a list of codenames for LTS releases. Codenames for future
+releases are subject to change.
```

Comment:
It just hit me that this is the same *repo* as `./schedule.json`. Could we at least mention that file or even have a link from there?

Reply:
also README.md here mentions the codenames and could link to CODENAMES.md